group_vars/all/main.yml

```diff
@@ -105,6 +117,10 @@ openshift_routes:
   - openshift/edxapp/route/cms.yml
   - openshift/edxapp/route/lms.yml
   - openshift/richie/route/richie.yml
+openshift_routes_prefixes:
+  - previous
+  - current
+  - next
```

Comment:
I'm skeptical that this should be configurable. Keep it simple and impose it?

Reply:
Ok.